hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/resource/RefreshIndexResourceIntTest.kt

```diff
@@ -0,0 +1,51 @@
+@file:Suppress("ClassName")
+
+package uk.gov.justice.digital.hmpps.prisonersearch.indexer.resource
+
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.times
+import uk.gov.justice.digital.hmpps.prisonersearch.common.model.IndexState
+import uk.gov.justice.digital.hmpps.prisonersearch.common.model.IndexStatus
+import uk.gov.justice.digital.hmpps.prisonersearch.common.model.SyncIndex
+import uk.gov.justice.digital.hmpps.prisonersearch.indexer.IntegrationTestBase
+
+class RefreshIndexResourceIntTest : IntegrationTestBase() {
+
+  @Test
+  fun `Refresh index - no differences`() {
+    indexStatusRepository.save(IndexStatus(currentIndex = SyncIndex.GREEN, currentIndexState = IndexState.COMPLETED))
+
+    webTestClient.put().uri("/refresh-index")
+      .headers(setAuthorisation(roles = listOf("ROLE_PRISONER_INDEX")))
+      .exchange()
+      .expectStatus().isAccepted
+
+//    await untilCallTo { indexSqsClient.countAllMessagesOnQueue(indexQueueUrl).get() } matches { it!! > 0 }
```

Comment:
at present we just log the index message rather than creating a message for each prisoner.  Will uncomment in future PRs when we start processing the refresh requests